scripts/helpers/check-attr.js

```diff
@@ -12,8 +13,21 @@ import _ from 'lodash';
  */
 export const checkAttr = (key, attributes, manifest, undefinedAllowed = false) => {
 
-	if (Object.prototype.hasOwnProperty.call(attributes, key)) {
-		return attributes[key];
+	// Check if there is prefix in the attributes object.
+	const prefix = attributes?.prefix;
+	let newKey = key;
+
+	// If there is no prefix return the key as it was.
+	// If there is a prefix, remove the attribute component name prefix and replace it with the new prefix.
+	if (typeof prefix !== 'undefined') {
```

Comment:
```suggestion
	if (prefix) {
```

By using `?.` you're sure that it will never be undeclared, just that it can be `undefined`, so no need for `typeof` checks.

Reply:
I have tested this and it is not working, sorry